src/main/groovy/com/cloudbees/plugins/flow/FlowDSL.groovy

```diff
@@ -1,8 +1,9 @@
 /*
  * The MIT License
  *
- * Copyright (c) 2013, CloudBees, Inc., Nicolas De Loof.
- *                     Cisco Systems, Inc., a California corporation
+ * Copyright (c) 2013-2015, CloudBees, Inc., Nicolas De Loof.
```

Comment:
Because it is basically impossible to separate the copyright ownership of the various pieces, I feel that a single date range and a list of the contributes makes more sense.  The original work was done in 2015, but I will update due to review changes.